CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - Damage Scaling now has a help description
 - Fixed the database module setting a global variable called `callback` which breaks addons such as PointShop2
 - Fixed voicechat keybinds being shown even if voice is disabled
+- Fixed typo in binoculars DataTable refactor
```

Comment:
```suggestion
- The binocular zoom now uses a DataTable that is not already used by its weaponbase
```

This describes what was fixed a bit better. This certainly was not only a typo :D